src/GraphQL.Serialization.Newtonsoft/GraphQL.Serialization.Newtonsoft.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>GraphQL.Serialization.Newtonsoft</RootNamespace>
```

Comment:
@joemcbride  GraphQL.Serialization.Newtonsoft or GraphQL.Serialization.NewtonsoftJson ?

Reply:
`GraphQL.Serialization.NewtonsoftJson` is better I think

Reply:
Even `GraphQL.NewtonsoftJson` would work for me.

Reply:
OK. So `GraphQL.NewtonsoftJson` and `GraphQL.SystemTextJson` for symmetry?

Reply:
👍 I like the shorter names.

Reply:
Library name and namespace has been changed